modules/dnn/src/net_impl_fuse.cpp

```diff
@@ -247,7 +247,7 @@ void Net::Impl::fuseLayers(const std::vector<LayerPin>& blobsToKeep_)
                     {
                         // fuse naryEltwise layer
                         // bias must already be computed to fuse => bias layer must appear before convolution
-                        if (biasLayerData->id < ld.id)
+                        if (biasLayerData->id < ld.id && biasLayerData->consumers.size() == 1)
```

Comment:
The fusion strategy wants to bypass the `biasLayerData`. But the strategy only works when its consumer size is 1.